src/kernel/arch/x86/syscalls.zig

```diff
@@ -3,12 +3,13 @@ const log = std.log.scoped(.x86_syscalls);
 const builtin = @import("builtin");
 const is_test = builtin.is_test;
 const build_options = @import("build_options");
-const arch = if (is_test) @import("../../../../test/mock/kernel/arch_mock.zig") else @import("arch.zig");
+const arch = if (is_test) @import("arch_mock") else @import("arch.zig");
```

Comment:
For the else case, would the import be `@import("arch")` instead of `@import("arch.zig")`
Or does this not work for arch file and is only used for the kernel files.
This is the same for all files.

Reply:
Fab, thanks for the clarification.